src/main.rs

```diff
@@ -146,7 +138,6 @@ fn main() {
             opt.permissions.to_vec(),
             (opt.write_id, opt.read_ids.clone(), opt.access_ids.clone()),
             minimum_tock_kernel_version,
-            add_trailing_padding,
```

Comment:
We had this as an argument because we don't necessarily _want_ to include this padding, but we do because it makes using TABs on cortex-m easier. In the future we may want to change when we add trailing padding. I think we should keep this argument.

Reply:
I moved this argument to inside elf_to_tbf function, since elf_file is moved to convert.rs:56~63. This argument (add_trailing_padding) is taken from elf::File.